docs/conf.py

```diff
@@ -384,6 +384,23 @@ def linkcode_resolve(domain, info):
 nbsphinx_execute = "never"
 # -- Options for LaTeX output ---------------------------------------------
 
+# -- Detect branch name -----------------------------------
+# Works on GitHub Actions, ReadTheDocs, and local fallback
+BRANCH = (
+    os.environ.get("GITHUB_REF_NAME")  # GitHub Actions
+    or os.environ.get("READTHEDOCS_VERSION")  # ReadTheDocs
+    or "main"  # fallback
+)
+
+nbsphinx_prolog = f"""
+.. raw:: html
+
+    <a href="https://mybinder.org/v2/gh/OceanParcels/Parcels/{BRANCH}?urlpath=lab/tree/docs/{{{{  env.doc2path(env.docname, base=None)  }}}}" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg"></a>
+    or view it <a href="https://github.com/OceanParcels/Parcels/blob/{BRANCH}/docs/{{{{  env.doc2path(env.docname, base=None)  }}}}" target="_blank">on GitHub</a>
```

Comment:
```suggestion
    Run this notebook in the cloud <a href="https://mybinder.org/v2/gh/OceanParcels/Parcels/{BRANCH}?urlpath=lab/tree/docs/{{{{  env.doc2path(env.docname, base=None)  }}}}" target="_blank"><img alt="Binder badge" src="https://mybinder.org/badge_logo.svg"></a>
    , or view it <a href="https://github.com/OceanParcels/Parcels/blob/{BRANCH}/docs/{{{{  env.doc2path(env.docname, base=None)  }}}}" target="_blank">on GitHub</a>
```

For those that don't know what binder is ;)